### PR TITLE
Dilpreet | Stage | PRod Bugs

### PR DESCRIPTION
**Pull request by dilpreetsinghofficial**
https://app.clickup.com/t/86cwfvwgt
https://app.clickup.com/t/86cwg7rwk
https://app.clickup.com/t/86cwfzrqt
https://app.clickup.com/t/86cwguu1q
https://app.clickup.com/t/86cwbx294
https://app.clickup.com/t/86cwahbm9
https://app.clickup.com/t/86cwdjz5w
https://app.clickup.com/t/86cwgv8qk
https://app.clickup.com/t/86cwh61ce

### CODE REVIEW

**Review by dvCodeWorld — DISMISSED** [review]

**Review by dvCodeWorld — DISMISSED** [review]

File: apps/web-giddh/src/app/add-company/add-company.component.ts [hunk 1]
```diff
@@ -1247,6 +1252,9 @@ export class AddCompanyComponent implements OnInit, AfterViewInit, OnDestroy {
             delete this.company.permission;
             delete this.company.creatorSuperAdmin;
         }
+        if (this.secondStepForm.value.businessNature === "Other") {
+            this.company.otherBusinessNature = this.secondStepForm.value.otherBusinessNature;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/add-company/add-company.component.ts:1256]: Either use controls to get value or this way. don't use both ways. follow 1 pattern only

File: apps/web-giddh/src/app/subscription/subscription.component.ts [hunk 2]
```diff
@@ -204,10 +204,12 @@ export class SubscriptionComponent implements OnInit, OnDestroy {
             }
         });
 
-        this.store.pipe(select(state => state.session.activeCompany), takeUntil(this.destroyed$)).subscribe(activeCompany => {
+        this.store.pipe(select(profile => profile.settings.profile), takeUntil(this.destroyed$)).subscribe((activeCompany) => {
             if (activeCompany) {
+                this.selectedCompany = null;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/subscription/subscription.component.ts]: what's this????

**Review by ravinderwalkover — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/subscription/subscription.component.ts (hunk 2 above)
Comment (dilpreetsinghofficial) [reply apps/web-giddh/src/app/subscription/subscription.component.ts]: @ravinderwalkover sir, for refresh proper response

File: apps/web-giddh/src/app/add-company/add-company.component.ts (hunk 1 above)
Comment (ankitdubeyoffice) [reply apps/web-giddh/src/app/add-company/add-company.component.ts:1256]: Done

File: apps/web-giddh/src/app/subscription/subscription.component.ts (hunk 2 above)
Comment (ravinderwalkover) [reply apps/web-giddh/src/app/subscription/subscription.component.ts]: Fix this

**Review by ravinderwalkover — APPROVED** [review]

**Pull request merged**